### ISSUE TITLE
create a transcriptome.fa

### ISSUE THREAD

**Issue by yuanjianwen-gif** [JiejunShi/BASAL#1]
Hi，nice tools! And, How to create a transcriptome.fa file? Thanks!

**Comment by yuanjianwen-gif** [comment]
Or just use the transcriptome fasta file from  gencode ？

**Comment by JiejunShi** [comment]
Exactly. We recommend to use the genome/transcriptome fasta files from gencode. The release versions we used can be found in our NAR paper.

**Comment by yuanjianwen-gif** [comment]
Thanks! Also, if need to remove pcr duplecates, when should  I  run this step, after 'basalkit.py mergeBAM' ?

**Comment by JiejunShi** [comment]
That's right.
